integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -0,0 +1,220 @@
+                    "executionId": "{$.cumulus_meta.execution_name}",
+                    "collectionShortname": "{$.meta.collection.name}",
+                    "collectionVersion": "{$.meta.collection.version}",
+                    "defaultBucketOverride": "{$.meta.collection.meta.orca.defaultBucketOverride}",
+                    "defaultStorageClassOverride": "{$.meta.collection.meta.orca.defaultStorageClassOverride}"
+                },
+                "exception": "None"
+                }
+
+            execution_info = boto3.client("stepfunctions").start_execution(
+                stateMachineArn=helpers.orca_copy_to_archive_step_function_arn,
+                input=json.dumps(copy_to_archive_input, indent=4),
+            )
+
+            step_function_results = helpers.get_state_machine_execution_results(
+                execution_info["executionArn"]
+            )
+
+            self.assertEqual(
+                "SUCCEEDED",
+                step_function_results["status"],
+                "Error occurred while starting step function.",
+            )
+            self.assertEqual(
+                expected_output,
+                json.loads(step_function_results["output"]),
+                "Expected step function output not returned.",
+            )
+            # verify that the objects exist in recovery bucket
+            try:
+                for key in [key_name_1, key_name_2]:
+                    head_object_output = boto3.client("s3").head_object(Bucket=recovery_bucket_name, Key=key)
+                    self.assertEqual(
+                        200,
+                        head_object_output["ResponseMetadata"]["HTTPStatusCode"],
+                        f" Error searching for object {key} in the {recovery_bucket_name}",
+                    )            
+            except Exception as ex:
+                return  ex
+
+            catalog_output = helpers.post_to_api(
+                my_session,
+                helpers.api_url + "/catalog/reconcile/",
+                data=json.dumps(
+                    {
+                        "pageIndex": 0,
+                        "granuleId": [granule_id_1, granule_id_2],
+                        "endTimestamp": int((time.time() + 5) * 1000),
+                    }
+                ),
+                headers={"Host": helpers.aws_api_name},
+            )
+            self.assertEqual(
+                200, catalog_output.status_code, "Error occurred while contacting API."
+            )
+            self.assertEqual(
+                {
+                    "anotherPage": False,
+                    "granules": [
+                        {   "id": granule_id_1,
+                            "collectionId": collection_name + "___" + collection_version,
+                            "createdAt": createdAt_time,
+                            "executionId": execution_id,
+                            "providerId": provider_id,
+                            "files": [],
```

Comment:
coube possibly be due to files being ingested already

Reply:
On retrospect, likely a race condition with the metadata update process.

Reply:
based on the ingest test, files are being copied to proper orca buckets by the workflow. When catalog is queried, we can see the granuleid and other parameters except the file.
So this looks like some bug in our ingest or API. I suggest we create a task to look into this for additional discussion.

https://bugs.earthdata.nasa.gov/browse/ORCA-631

Reply:
I cleaned out the ORCA catalog and the bucket, and I see the file data coming through perfectly fine
-                'files': [],
+                'files': [{'cumulusArchiveLocation': 'orca-sandbox-s3-provider',
+                           'hash': None,
+                           'hashType': None,
...
This is why randomizing these values or performing proper cleanup is vital.
Even if it really wasn't coming through, we should not create a faulty test for the sake of having a test that passes.

Reply:
Here is the full output I got from the API Gateway:
` {"anotherPage":false,"granules":[{"providerId":"3dd94c6a-31bc-4039-a1cd-c86e6dcc10f3","collectionId":"f75c1a77-5359-49dd-a5c0-fea127d93c2b___34267030-3f5d-4a91-81d1-cebbeda2a6ff","id":"3776ac04-42b7-4a92-9b80-95f11ef89a2e","createdAt":1675443016038,"executionId":"c69a52af-cd61-415e-8555-072e53d16932","ingestDate":1675443013533,"lastUpdate":1675443013533,"files":[{"name":"ancillary_data_input_forcing_ECCO_V4r4.tar.gz","cumulusArchiveLocation":"orca-sandbox-s3-provider","orcaArchiveLocation":"doctest-orca-primary","keyPath":"PODAAC/SWOT/ancillary_data_input_forcing_ECCO_V4r4.tar.gz","sizeBytes":205640819682,"hash":null,"hashType":null,"storageClass":"GLACIER","version":"TcWopqa0O9YGrKwgAujlLs2azSMJYX2y"}]}]}`

Reply:
good to know it is working.

Reply:
Confirmed that the test only works if the catalog is cleaned between runs. Add to docs.

Reply:
added to integration test doc.